packages/analytics-v1.1/src/utils/constants.js

```diff
@@ -20,7 +20,7 @@ const DEST_SDK_BASE_URL = `${SDK_CDN_BASE_URL}/${CDN_ARCH_VERSION_DIR}/${CDN_INT
 const DEFAULT_DATA_PLANE_EVENTS_BUFFER_TIMEOUT_MS = 10000;
 const INTG_SUFFIX = '_RS';
 const POLYFILL_URL =
-  'https://polyfill.io/v3/polyfill.min.js?features=Number.isNaN%2CURL%2CArray.prototype.find%2CArray.prototype.includes%2CPromise%2CString.prototype.endsWith%2CString.prototype.includes%2CString.prototype.startsWith%2CObject.entries%2CObject.values%2CElement.prototype.dataset%2CString.prototype.replaceAll%2CTextDecoder%2Uint8Array';
+  'https://polyfill.io/v3/polyfill.min.js?features=Number.isNaN%2CURL%2CArray.prototype.find%2CArray.prototype.includes%2CPromise%2CString.prototype.endsWith%2CString.prototype.includes%2CString.prototype.startsWith%2CObject.entries%2CObject.values%2CElement.prototype.dataset%2CString.prototype.replaceAll%2CTextDecoder%2Cnavigator.sendBeacon';
```

Comment:
`Uint8Array` is not required anymore?

Reply:
It is not supported anymore

Reply:
@MoumitaM, we need to check whether these polyfill query strings are valid individually. For example, `TextDecoder` is not supported.